virtualbox/vbm.go

```diff
@@ -99,6 +100,11 @@ func getHostOnlyNetworkInterface(mc *driver.MachineConfig) (string, error) {
 
 	for _, n := range nets {
 		if dhcp, ok := dhcps[n.NetworkName]; ok {
+            fmt.Printf("IPv4 / DHCP IP: %s == %s\n", dhcp.IPv4.IP, mc.DHCPIP)
```

Comment:
It's probably a bigger issue to tackle. Maybe even an issue with VirtualBox 5, I don't think the creation of host-only networks works anymore. I just reported the same issue for docker-machine: https://github.com/docker/machine/issues/1521